pkg/api/whereabouts.cni.cncf.io/v1alpha1/ippool_types.go

```diff
@@ -24,6 +24,7 @@ func (i IPPool) ParseCIDR() (net.IP, *net.IPNet, error) {
 type IPAllocation struct {
 	ContainerID string `json:"id"`
 	PodRef      string `json:"podref"`
+	Interface   string `json:"interface"`
```

Comment:
Shall we use ifName to align with the CNI parameter?

Reply:
same as above